app/services/rhapsody_realities.js

```diff
@@ -73,7 +74,7 @@ class RhapsodyService {
             .then(count => {
                 pages.total = count[0].count;
                 return new Rhapsody().query(qb => {
-                    qb.limit(pages.limit).offset(pages.offset);
+                    qb.where(knex.raw('to_char(date, \'MM\')'), filterBy).limit(pages.limit).offset(pages.offset);
```

Comment:
you need the whole thing, i.e month and year, otherwise you will be returning for all the say Octobers of all the year in the system

month and year combination is what you want to ensure result is unique